.pre-commit-config.yaml

```diff
@@ -19,13 +19,12 @@ repos:
   hooks:
   - id: isort
     args:
+    # Keep in sync with pyproject.toml
+    - --force-single-line-imports
     - --honor-noqa
-    exclude: |
-      (?x)
-      ^
-        .*
-      $
-    stages: ["manual"]
+    - --lines-between-types=1
+    - --profile=black
```

Comment:
I don't like this implicit thing that hides what this really means behind the internals. Look into the example above to see how it is possible to be more verbose about the settings intentions.
Another point is that using a proper per-tool annotated config allows to track the motivation and expectation behind the settings with obscured names.

Reply:
I take it you would prefer to have the individual lines that make up the black profile listed?  The downside here is that the burden of keeping our isort configuration compatible with black falls us.  I don't have an issue with this, but will want to add a comment that noting the configuration needs to be kept in sync with the black profile, tracking it's source.

Reply:
I agree with @webknjaz here, we should have zero arguments for isort inside pre-commit config. That is more important than you think because we want to ensure that even if you call isort from outside it will have the same behavior. vscode has the ability to run isort on save and you don't want it to sort files differently than pre-commit.

In most cases any arguments/config in pre-commit is a potential issue (very few exceptions).

Reply:
> I take it you would prefer to have the individual lines that make up the black profile listed?

With our improvements, though. OTOH, it would probably be fine if we keep profile + things that are important to keep explicit.

Reply:
config remove from pre-commit entirely, will keep the profile for now, until it becomes a problem, thx